pgx-pg-sys/build.rs

```diff
@@ -536,14 +536,26 @@ fn run_bindgen(pg_config: &PgConfig, include_h: &PathBuf) -> eyre::Result<syn::F
         .header(include_h.display().to_string())
         .clang_arg(&format!("-I{}", includedir_server.display()))
         .parse_callbacks(Box::new(PgxOverrides::default()))
+        .allowlist_file(".*confdefs.h")
+        .allowlist_file(".*([fF]uncs|mgr).h")
+        .allowlist_file(".*htup(_details)?.h")
+        .allowlist_file(".*syscache.h")
+        .allowlist_file(".*mcxt.h")
+        .allowlist_file(".*(storage|catalog|access|commands|executor|adt|optimizer|rewrite|postmaster|tcop|replication|nodes|postgres|parse|pg_|item|heap).*")
+        .allowlist_function("memcpy")
+        .allowlist_var("SIG.*")
         .blocklist_type("Datum") // manually wrapping datum types for correctness
         .blocklist_type("NullableDatum")
         .blocklist_function("varsize_any") // pgx converts the VARSIZE_ANY macro, so we don't want to also have this function, which is in heaptuple.c
         .blocklist_function("query_tree_walker")
         .blocklist_function("expression_tree_walker")
-        .blocklist_function("sigsetjmp")
-        .blocklist_function("siglongjmp")
+        .blocklist_function(".*setjmp")
+        .blocklist_function(".*longjmp")
         .blocklist_function("pg_re_throw")
+        .blocklist_item("CONFIGURE_ARGS") // configuration during build is hopefully irrelevant
+        .blocklist_item("_*(HAVE|have)_.*") // these are for C's use
+        .blocklist_item("__[A-Z].*") // these are reserved
```

Comment:
These names are seriously special super duper reserved names we should never be looking at.

Reply:
Well, there's a number of `_[A-Z]` and `__.*` that seem to actually be in use in our bindings, so I didn't omit all of them, even though I wanted to. I think Postgres uses `_[A-Z]` in its names even though it's not supposed to. But we can add a blocklist for `__darwin.*`?